### ISSUE TITLE
`makepkg` in perl fails one of the sha256sum checks

### ISSUE THREAD

**Issue by kubrat** [msys2/MSYS2-packages#1068]
```
==> Validating source files with sha256sums...
    perl-5.24.1.tar.bz2 ... Passed
    perlbin.sh ... Passed
    perlbin.csh ... Passed
    perlbin.fish ... Passed
    perl-binary-module-dependency-1.template ... FAILED
    perl.cygwin-auto-image-base.patch ... Passed
    perl.cygwin-hints.patch ... Passed
    perl.cygwin-Configure-libpth.patch ... Passed
    perl.cygwin-Configure-libsearch.patch ... Passed
    perl-5.22.0-msys2.patch ... Passed
    Compress-Raw-Zlib-2.071-Adapt-to-zlib-1.2.11.patch ... Passed
    Compress-Raw-Zlib-2.071-Conform-to-C90.patch ... Passed
    perl-5.24.0-assertion-failure-in-.-or-0-x-0.patch ... Passed
    regen-skip.diff ... Passed
    test-suite-without-dot.diff ... Passed
    regen-path.patch ... Passed
    Split-APItest-t-handy.t.patch ... Passed
    Remove-support-for-Splint-static.patch ... Passed
==> ERROR: One or more files did not pass the validity check!
```

**Comment by cbrt64** [comment]
2c882a9dec49fe23bff1d7687cd3c98e8422042e removed DOS EOLs from `perl-binary-module-dependency-1.template`, but didn't update the checksum in PKGBUILD.

**Comment by cbrt64** [comment]
This can be closed, the build script checksum was fixed soon after with 02fc474.
